src/models/parsimonious.py

```diff
@@ -21,10 +21,15 @@ def train(self) -> None:
     def save_model(self) -> None:
         print('Move on! Nothing to save here!')
 
-    def predict(self) -> Tuple[Dict[str, Dict[str, np.ndarray]], Dict[str, np.ndarray]]:
-
-        test_arrays_loader = DataLoader(data_path=self.data_path, batch_file_size=self.batch_size,
-                                        shuffle_data=False, mode='test', normalize=False)
+    def predict(
+        self,
+        experiment: str = 'one_month_forecast'
```

Comment:
Howcome this is passed as an argument to `predict`? Would it make more sense as a class attribute?

Could it even be hardcoded given the nature of this "model"?

Reply:
thanks for finding these mate!

Reply:
great spot

Reply:
Re: hardcoded - it could be used as a baseline for any model? So even for the `nowcast` experiment persistence would still be a reasonable baseline to compare against?